### PR TITLE
initial edits for results

### PR DESCRIPTION
**Pull request by cgreene**
I'm starting through the results. I had a couple questions/suggestions (as comments) in addition to these changes. I'll try to keep working on this branch through the first section, but please provide feedback and I'll incorporate it as I go. Thanks!

### CODE REVIEW

File: content/04.results.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
-| avg. document length        | 312.10      | 242.96        | 38.89         | 
-| avg. sentence length        | 22.71       | 21.46         | 19.89         | 
-| negatives                  | 1,148,382   | 24,928,801    | 7,272,401     | 
-| coordinating conjunctions  | 14,295,736  | 307,082,313   | 38,730,053    | 
-| coordinating conjunctions% | 3.40%       | 3.57%         | 3.18%         | 
-| pronouns                   | 4,604,432   | 74,994,125    | 46,712,553    | 
-| pronouns%                  | 1.09%        | 0.87%         | 3.83%         | 
-| passives                   | 15,012,441  | 342,407,363   | 19,472,053    | 
-| passive%                   | 3.57%       | 3.98%         | 1.60%         | 
+We found that nearly all preprints were categorized as new results, which is consistent with previous findings [@doi:10.1001/jama.2017.21168]. <!-- I cannot find this assertion or these numbers in this paper - where did you see it? -->
```
Comment (danich1) [inline content/04.results.md:17]: I originally had a figure that showed majority of new results were in the bioRxiv repository, but Larry told me to remove the figure (as it was a waste of space) and just state majority of preprints were new results.

Comment (cgreene) [reply content/04.results.md:17]: Oh - I meant in the paper that the reference goes to

Comment (danich1) [reply content/04.results.md:17]: There is a table within the letter that shows 7424 preprints with new results compared to the other categories (221 confirmatory and 105 contradictory).  Table is called: Descriptive Characteristics of Preprints in bioRxiv

Comment (cgreene) [reply content/04.results.md:17]: OMG the table doesn't show up in the web view but does in the PDF 🤯  😂

File: content/04.results.md [hunk 2]
```diff
@@ -1,38 +1,39 @@
...
-| avg. document length        | 312.10      | 242.96        | 38.89         | 
-| avg. sentence length        | 22.71       | 21.46         | 19.89         | 
-| negatives                  | 1,148,382   | 24,928,801    | 7,272,401     | 
-| coordinating conjunctions  | 14,295,736  | 307,082,313   | 38,730,053    | 
-| coordinating conjunctions% | 3.40%       | 3.57%         | 3.18%         | 
-| pronouns                   | 4,604,432   | 74,994,125    | 46,712,553    | 
-| pronouns%                  | 1.09%        | 0.87%         | 3.83%         | 
-| passives                   | 15,012,441  | 342,407,363   | 19,472,053    | 
-| passive%                   | 3.57%       | 3.98%         | 1.60%         | 
+We found that nearly all preprints (XY.Z%) were categorized as new results, which is consistent with reports on a smaller set [@doi:10.1001/jama.2017.21168].
```
Comment (danich1) [inline content/04.results.md:17]:
```suggestion
We found that nearly all preprints (97.5%) were categorized as new results, which is consistent with reports on a smaller set [@doi:10.1001/jama.2017.21168].
```

**Pull request merged**